readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -83,7 +83,7 @@ def append_conf(self, **kwargs):
             display_bitbucket = True
 
         rtd_ctx = Context({
-            'current_version': self.version.slug,
+            'current_version': self.version.verbose_name,
```

Comment:
Hmm, this should probably use the slug at some point in the future, but this is likely correct for backwards compat.